<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWidget</class>
 <widget class="QWidget" name="MapWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>327</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="opaqueResize">
          <bool>false</bool>
         </property>
         <property name="handleWidth">
          <number>5</number>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <property name="spacing" stdset="0">
          <number>2</number>
         </property>
         <property name="leftMargin" stdset="0">
          <number>2</number>
         </property>
         <property name="topMargin" stdset="0">
          <number>2</number>
         </property>
         <property name="rightMargin" stdset="0">
          <number>2</number>
         </property>
         <property name="bottomMargin" stdset="0">
          <number>2</number>
         </property>
         <widget class="FlowView" name="flowView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
         </widget>
         <widget class="QWidget" name="sidebar" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignRight|Qt::AlignVCenter">
              <widget class="QToolButton" name="toolButton">
               <property name="text">
                <string>Hide</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="arrowType">
                <enum>Qt::RightArrow</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="ScriptEdit" name="scriptEdit"/>
           </item>
           <item>
            <widget class="QListWidget" name="listRooms">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolBar" name="toolBar">
             <property name="floatable">
              <bool>false</bool>
             </property>
             <addaction name="actionAttachRoom"/>
             <addaction name="actionDetachRoom"/>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPreview">
      <attribute name="title">
       <string>Preview</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="NovelTeaWidget" name="preview" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProperties">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="PropertyEditor" name="propertyEditor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionChangeRoomName">
   <property name="text">
    <string>Change Room Name</string>
   </property>
  </action>
  <action name="actionEditScript">
   <property name="text">
    <string>Edit Script</string>
   </property>
  </action>
  <action name="actionAttachRoom">
   <property name="icon">
    <iconset theme="add">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Attach Room</string>
   </property>
  </action>
  <action name="actionDetachRoom">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="remove">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PropertyEditor</class>
   <extends>QWidget</extends>
   <header>Widgets/PropertyEditor.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FlowView</class>
   <extends>QGraphicsView</extends>
   <header>Widgets/Map/FlowView.hpp</header>
  </customwidget>
  <customwidget>
   <class>NovelTeaWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/NovelTeaWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScriptEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>Widgets/ScriptEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
